composer.json

```diff
@@ -64,14 +64,17 @@
         "knplabs/knp-gaufrette-bundle": "~0.1",
         "symfony-cmf/routing-bundle": "~2.0",
         "fpn/doctrine-extensions-taggable": "~0.9",
+        "twig/twig": "^1.12|^2.0",
         "sensio/generator-bundle": "~3.0",
         "twig/twig": "^1.36|^2.6",
         "twig/extensions": "~1.0",
         "egulias/email-validator": "^1.2.8|^2.0",
         "box/spout": "^2.5",
         "ruflin/elastica": "^5.1|^6.0",
         "behat/transliterator": "~1.2",
-        "defuse/php-encryption": "v2.1.0"
+        "defuse/php-encryption": "v2.1.0",
+        "symfony/maker-bundle": "^1.9",
```

Comment:
Do we need the maker bundle now? I don't think we use a maker in the current code